packages/app-project/src/helpers/fetchWorkflowsHelper/fetchWorkflowsHelper.js

```diff
@@ -20,7 +20,8 @@ async function fetchWorkflowData(workflows, env) {
   }
 }
 
-async function fetchSingleWorkflow(workflowID, env) {
+/* Also used as a fallback fetch when leveling-up handles retired workflows */
+export async function fetchSingleWorkflow(workflowID, env) {
   const { headers, host } = getServerSideAPIHost(env)
```

Comment:
This is a server-side data-fetching function, so it probably won't work in a browser, at least as written.

Reply:
It's probably fine in local development, but won't work in the browser once the code is deployed to kubernetes.
https://github.com/zooniverse/front-end-monorepo/blob/6865656a84af4dd7a2a043f1434266e83a18aa3e/packages/app-project/src/helpers/getServerSideAPIHost/getServerSideAPIHost.js#L7-L10

Reply:
Oh that's true. I'll have to write a custom fetch single workflow function instead.